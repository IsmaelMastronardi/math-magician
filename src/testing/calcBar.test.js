import { render } from '@testing-library/react';
import ResultBar from '../components/CalculatorBar';

describe('Renders a result bar', () => {
  test('render', () => {
    const { bar } = render(<ResultBar item="0" />);
    expect(bar).toMatchSnapshot();
  });
  test('get value from result bar', () => {
    const { getByTestId } = render(<ResultBar item="0" />);
    const value = getByTestId('resultBarTestId').textContent;
    expect(value).toBe('0');
  });
});
