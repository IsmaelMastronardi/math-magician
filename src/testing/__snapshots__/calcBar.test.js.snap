// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders a result bar get value from result bar 1`] = `undefined`;

exports[`Renders a result bar render 1`] = `undefined`;
